albumentations/augmentations/functional.py

```diff
@@ -220,6 +220,17 @@ def random_crop(img, crop_height, crop_width, h_start, w_start):
     img = img[y1:y2, x1:x2]
     return img
 
+def simple_crop_coordinates(img, x_min, x_max, y_min, y_max):
```

Comment:
crop_image_by_coordinates

consider adding same method for bounding boxes

Reply:
"consider adding same method for bounding boxes" - what does this method should do?

Reply:
crop bounding boxes by same "crop_box". new feature should work for all existing targets